src/test/java/seedu/budgetbuddy/ExpenseListTest.java

```diff
@@ -92,4 +99,34 @@ public void testDeleteExpense_indexOutOfBounds() {
         expenseList.deleteExpense(initialSize + 1); // Trying to delete with index out of bounds
         assertEquals(initialSize, expenseList.getExpenses().size()); // Size should remain the same
     }
+
+    @Test
+    public void testGetExpenseInsights() {
```

Comment:
The format of the method seems different. Should it be getExpenseSights_validSetOfExpenses_printsValidOutput() instead ?